app/services/discovery_fact_importer.rb

```diff
@@ -0,0 +1,9 @@
+class DiscoveryFactImporter < StructuredFactImporter
+  def self.authorized_smart_proxy_features
+    'Discovery'.freeze
```

Comment:
Will this work or is this even necessary? The proxy running on FDI is typically not registered in Foreman, but I have no idea how authorization works in this case of facts upload. Perhaps we're skipping it entirely?

Reply:
I just copied a `PuppetFactImporter`, so that's the reason, why this method has been created. I tried that without this method and `DiscoveryFactImporter` working properly. So I deleted this method and updated PR.